vendor.conf

```diff
@@ -145,7 +146,7 @@ github.com/klauspost/compress                       a3b7545c88eea469c2246bee0e6c
 github.com/pelletier/go-toml                        65ca8064882c8c308e5c804c5d5443d409e0738c # v1.8.1
 
 # cluster
-github.com/docker/swarmkit                          5a5494a9a7b408b790533a5e4e1cb43ca1c32aad
+github.com/docker/swarmkit                          60d87cb7cdb070801ec550d7f4d7dc1210fb7e9f git://github.com/cpuguy83/swarmkit.git
```

Comment:
:cowboy_hat_face:

Reply:
Yeah, we'll need to update these projects once merged.